src/makeCalculator.js

```diff
@@ -38,6 +38,41 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, aNumber) {
+      callback(aNumber);
+
+      return this;
+    },
+
+    add(a) {
+      calculator.result = calculator.result + a;
```

Comment:
You can shorten it. Same goes to all methods
```suggestion
      calculator.result += a;
```